logstash-core/src/main/java/org/logstash/config/ir/imperative/ComposedParallelStatement.java

```diff
@@ -0,0 +1,32 @@
+package org.logstash.config.ir.imperative;
```

Comment:
@jordansissel we need parallel statements to represent outputs which are unordered, as opposed to filters, which are ordered.

In the future we'll need to make this explicit. Users will then have the option of declaring whether filter or output order matters. You might want ordered outputs or unordered filters, or sub-parts of each to act differently, hence the requirement.